index.html

```diff
@@ -16,25 +16,13 @@
   <body>
     <!--header-->
     <header class="header">
-<<<<<<< HEAD
       <img class="logo" src="./logo.svg" alt="LOGO" />
```

Comment:
- [ ] let's make sure you import the logo well.
- [ ] Please, use <a> tag for your logo because this link will point to the header of your site